packages/web-config-server/README.md

```diff
@@ -74,27 +74,6 @@ API connect for Tupaia App server
 
 Deals with oauth2 authentication and request from dhis2
 
-## Exporting charts and Lambda
```

Comment:
Unrelated cleanup, this is outdated documentation about a system that we no longer use